salt/daemons/masterapi.py

```diff
@@ -661,6 +661,13 @@ def _file_recv(self, load):
             log.error('Invalid file pointer: load[loc] < 0')
             return False
 
+        if 'size' in load and load['size'] > file_recv_max_size:
```

Comment:
this is an anti-pattern - doing 'foo' in dict and dict['foo'], as described here:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/not_using_get_to_return_a_default_value_from_a_dictionary.html

You should be using dict.get(), as in: if load.get('size', 0) > file_recv_max_size:

Reply:
Thanks, fixed.